components/PageTitle/PageTitle.tsx

```diff
@@ -7,11 +7,12 @@ interface Props {
     title: string;
     subtitle?: ReactNode;
     className?: string;
+    bgClassName?: string;
```

Comment:
Let's try to not shorten the class names like this. `backgroundClassName` is not that long after all :)

Reply:
Also, I was under the impression, that we want to remove the bg color and border from ALL Page Title blocks. Then this extra class name will not be needed. Please correct me if I'm mistaken.

Reply:
Checked the other pages and all page titles are having this same design now, will update and remove the extra classes